core/src/main/java/io/questdb/griffin/model/CreateTableModel.java

```diff
@@ -243,6 +249,12 @@ public void setTimestamp(ExpressionNode timestamp) {
         this.timestamp = timestamp;
     }
 
+    public void setVolumeAlias(CharSequence volumeAlias) {
+        // set if the create table statement contains IN VOLUME 'volumeAlias'.
+        // volumePath will be resolved by the compiler
+        this.volumeAlias = volumeAlias;
```

Comment:
`CharSequence`s are usually transient, so it's safer to store a copy of them:
```suggestion
        this.volumeAlias = Chars.toString(volumeAlias);
```

Reply:
good spot!